### PR TITLE
Add a hint for users about using the canonical URL

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

This pull request (PR) adds a hint for users about using the canonical URL.

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/marketing/url-rewrite-configuration.html

## Links to Magento source code or PRs

- https://developers.google.com/search/docs/advanced/crawling/consolidate-duplicate-urls
- https://support.google.com/webmasters/answer/9012289#google-selected-canonical

### CODE REVIEW

File: src/marketing/url-rewrite-configuration.md [hunk 1]
```diff
@@ -26,3 +26,21 @@ URL with Web Server Rewrite
 1. Set **Use Web Server Rewrites** to your preference.
 
 1. When complete, click <span class="btn">Save Config</span>.
+
+## Use canonical URLs
```
Comment (dobooth) [inline src/marketing/url-rewrite-configuration.md:30]:
```suggestion
## Canonical URLs
```

File: src/marketing/url-rewrite-configuration.md [hunk 2]
```diff
@@ -26,3 +26,21 @@ URL with Web Server Rewrite
...
+
+It is imperative that each of your web pages only has one separate URL for SEO consolidation.
+
+If you have a single page accessible by multiple URLs, or different pages with similar content, Google sees these as duplicate versions of the same page. Google will choose one URL as the canonical version and crawl that, and all other URLs will be considered duplicate URLs and crawled less often.
+
+If you don't explicitly tell Google which URL is canonical, Google will make the choice for you, or might consider them both of equal weight, which might lead to unwanted behavior, run the risk of an ineffective crawl budget and low distributed backlinks.
```
Comment (dobooth) [inline src/marketing/url-rewrite-configuration.md:36]: This is a long sentence. Break it up into two.

**Review by dobooth — CHANGES_REQUESTED** [review]

**Comment by EvheniiaKolkhiieva** [comment]
Thank you for review @dobooth,
I have implemented your suggestions, could you please approve this PR?

File: src/marketing/url-rewrite-configuration.md (hunk 2 above)
Comment (EvheniiaKolkhiieva) [reply src/marketing/url-rewrite-configuration.md]: Done ✅

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the pull request successfully imported.

**Pull request merged**